src/navigation/RootStackNavigator.types.ts

```diff
@@ -15,3 +17,5 @@ export type RootStackParamList = {
   UpdateAccommodation?: { accommodation: Accommodation };
   AddAccommodationImage?: any;
 };
+
+export type Props = StackScreenProps<RootStackParamList, 'VerifyEmail'>;
```

Comment:
The same here. Why just `Props`?

It is not a common type of props of any component. Why is it placed here along with Navigatior types?

Usually, props types are placed along with components that are using this type.